src/core/dbus-unit.c

```diff
@@ -561,6 +587,7 @@ int bus_unit_method_kill(sd_bus_message *message, void *userdata, sd_bus_error *
                         u,
                         "kill",
                         N_("Authentication is required to send a UNIX signal to the processes of '$(unit)'."),
+                        /* transient_owner = */ false, /* Sending arbitrary signal is not allowed even for owners */
```

Comment:
This is unexpected. I'd think that the owner is allowed to do anything with the unit, incl. send send signals, freeze or thaw, etc.

Reply:
It's possible to relax this at a later time. But doing arbitrary operation to the process itself sounds dangerous and you can't be too careful when crossing privilege boundaries.